cli/compose/convert/service_test.go

```diff
@@ -623,3 +624,48 @@ func TestConvertUpdateConfigParallelism(t *testing.T) {
 	})
 	assert.Check(t, is.Equal(parallel, updateConfig.Parallelism))
 }
+
+func TestConvertServiceCapAddAndCapDrop(t *testing.T) {
+	service := composetypes.ServiceConfig{
+		CapAdd: []string{
+			"SYS_NICE",
+			"CAP_NET_ADMIN",
+		},
+		CapDrop: []string{
+			"CHOWN",
+			"DAC_OVERRIDE",
+			"CAP_FSETID",
+			"CAP_FOWNER",
+		},
+	}
+
+	expected := []string{
+		"CAP_MKNOD",
+		"CAP_NET_RAW",
+		"CAP_SETGID",
+		"CAP_SETUID",
+		"CAP_SETFCAP",
+		"CAP_SETPCAP",
+		"CAP_NET_BIND_SERVICE",
+		"CAP_SYS_CHROOT",
+		"CAP_KILL",
+		"CAP_AUDIT_WRITE",
+		"CAP_SYS_NICE",
+		"CAP_NET_ADMIN",
+	}
+
+	result, err := Service("1.41", Namespace{name: "foo"}, service, nil, nil, nil, nil)
+	assert.NilError(t, err)
+	assert.Check(t, is.DeepEqual(result.TaskTemplate.ContainerSpec.Capabilities, expected))
+}
+
+func TestConvertServicePrivileged(t *testing.T) {
+	service := composetypes.ServiceConfig{
+		Privileged: true,
+	}
+
+	expected := caps.GetAllCapabilities()
+	result, err := Service("1.41", Namespace{name: "foo"}, service, nil, nil, nil, nil)
```

Comment:
If you pulled 1.41 our into a const or var then it could self-describe what it is/does. I assume it means version?

Reply:
API version 1.41 is first one which contains ability to define Capabilities list so that one is called on here: https://github.com/docker/cli/blob/968ce1ae4d45722c6ae70aa1dff6ee28d88e976a/cli/compose/convert/service.go#L58-L60

Reply:
Added now also another test with older API version.